velox/connectors/Connector.h

```diff
@@ -129,6 +129,9 @@ FOLLY_ALWAYS_INLINE std::ostream& operator<<(
 /// Return a commit strategy of the given string encoding.
 CommitStrategy stringToCommitStrategy(const std::string& strategy);
 
+/// This class is responsible for writing data received from table writer
```

Comment:
Drop "This class is responsible" for readability.

```
Writes data ...
```

Reply:
Updated. Thanks!